src/api/item.js

```diff
@@ -32,4 +32,16 @@ app.get('/all', (req, res) => {
     });
 });
 
+app.delete('/', (req, res) => {
+    addAction('DELETE_ITEM', {
+        itemAddress: req.query.itemAddress
+    }).then(() => {
+        res.successJson({
+            items: ItemStore.getItems()
+        });
+    }).catch(e => {
+        res.failureJson(e);
```

Comment:
This needs to be `res.failureJson(e.message)`, otherwise the returned JSON would be `{"status": "failure"}`. See https://github.com/TheFourFifths/consus/blob/dev/src/lib/api.js#L27.